CHANGELOG.md

```diff
@@ -8,13 +8,40 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 For compatibility information see `govukFrontendVersion` and `hmrcFrontendVersion` in
 [LibDependencies](project/LibDependencies.scala)
 
-## [3.30.0] - 2022-10-03
+## [3.33.0] - 2022-10-24
+
+### Changed
+
+- Fixed up the changelog, we got confused with some build changes for services that potentially skip creating a new version if you've only changed non-functional code. That behaviour doesn't apply to libraries and so some changes to the docs which didn't have an entry in the changelog caused our versions in the changelog to drift from the actual release numbers
+
+### Compatible with
+
+- [hmrc/hmrc-frontend v5.11.1](https://github.com/hmrc/hmrc-frontend/releases/tag/v5.11.1)
+- [alphagov/govuk-frontend v4.3.1](https://github.com/alphagov/govuk-frontend/releases/tag/v4.3.1)
+
+## [3.32.0] - 2022-10-07
```

Comment:
deliberately updated the date on this, original pr was 3rd but it was actually merged on 7th - because I didn't want the new entry below for my readme update on the 4th to have v32 showing in the changelog as having been released before it